db/model.js

```diff
@@ -16,4 +16,22 @@ const fetchReviews = () => {
     })
 }
 
-module.exports = { fetchCategories, fetchReviews }
\ No newline at end of file
+const fetchReviewById = (id) => {
+    const parsedId = parseInt(id)
+    
+    if (id > 24) {
+        return Promise.reject({status: 404, msg: "invalid review ID"})
+    };
+
+    if (isNaN(parsedId) === true) {
+        return Promise.reject({status: 400, msg: "invalid data type"})
+    }
```

Comment:
With this one is there a way we could rely on this error going to our controller, passing it to our catch block using `next` and then pass it to our error handlers in app. Try console logging this error within your error handling functions in app and see if it has a psql error code you can use to then return the message you want.